includes/widgets/icon-box.php

```diff
@@ -152,16 +152,14 @@ protected function register_controls() {
 		$this->add_control(
 			'description_text',
 			[
-				'label' => '',
-				'type' => Controls_Manager::TEXTAREA,
+				'label' => esc_html__( 'Description', 'elementor' ),
+				'type' => Controls_Manager::WYSIWYG,
```

Comment:
Revert

Reply:
It's a pity that WYSIWYG has not been accepted, it may have the potential to break existing sites, indeed.